components/PeerReviews/PeerReviewInviteModal.tsx

```diff
@@ -0,0 +1,139 @@
+import { ReactElement, useRef, useState } from "react";
+import { css, StyleSheet } from "aphrodite";
+import { PeerReviewRequest } from "~/config/types/peerReview";
+import BaseModal from "~/components/Modals/BaseModal";
+import FormInput from "~/components/Form/FormInput";
+import Button from "~/components/Form/Button";
+import { inviteReviewer } from "./config/PeerReviewInviteAPI";
+import { MessageActions } from "~/redux/message";
+import { connect } from "react-redux";
+import Loader from "~/components/Loader/Loader";
+import colors from "~/config/themes/colors";
+
+type Props = {
+  peerReviewRequest: PeerReviewRequest;
+  fetchReviewRequests: Function;
+  isOpen: Boolean;
+  closeModal: Function;
+  setMessage: Function;
+  showMessage: Function;
+};
+
+function PeerReviewRequestModal({
+  peerReviewRequest,
+  fetchReviewRequests,
+  isOpen,
+  closeModal,
+  setMessage,
+  showMessage,
```

Comment:
same here. alphabetize if you can

Reply:
hmm how do you handle optional params? they typically go at the end of param list

Reply:
I typically list params in order of priority - the first one is the most important one.